src/curve_fitting/model.clj

```diff
@@ -26,37 +24,18 @@
       (reduce + (map (gen [n] (* (nth coeffs n) (expt x n)))
                      (range degree))))))
 
-(define outlier?
-  (gen []
-    (and outliers-enabled
-         (flip 0.1))))
-
 (define add-noise-to-curve
   (gen [curve]
+    ;; Outliers are disabled by default. Enable with an intervention trace.
+    (define outliers-enabled? (flip 1)) ; Are outliers enabled?
```

Comment:
Without an intervention one way or the other, won't this enable outliers half the time?

Reply:
The argument to `flip` is the probability that it returns `true`, so outliers should be enabled every time if no intervention trace is provided.

Reply:
I should update that comment.